rewatch/src/watcher.rs

```diff
@@ -78,10 +79,10 @@ async fn async_watch(
         plain_output,
         warn_error,
     }: AsyncWatchArgs<'_>,
-) -> notify::Result<()> {
+) -> Result<()> {
     let mut build_state: build::build_types::BuildCommandState =
         build::initialize_build(None, filter, show_progress, path, plain_output, warn_error)
-            .expect("Can't initialize build");
+            .with_context(|| "Could not initialize build")?;
```

Comment:
Message changed for consistency with the `build` command and other error messages.